src/main/scala/it/pagopa/interop/catalogprocess/api/impl/ProcessApiServiceImpl.scala

```diff
@@ -478,6 +490,13 @@ final case class ProcessApiServiceImpl(
         .getTenant(eservice.producerId)
         .flatMap(_.selfcareId.toFuture(MissingSelfcareId))
     )
+  */
+
+  def getGetEserviceAndTenant(eservice: client.model.EService)(implicit contexts: Seq[(String, String)])
+  : Future[(client.model.EService, TenantManagementDependency.Tenant)] =
+    eservice
+    .pure[Future]
+    .mproduct(eservice => tenantManagementService.getTenant(eservice.producerId))
```

Comment:
Can't be refactored to something similar to 👇 ?
```suggestion
    tenantManagementService.getTenant(eservice.producerId).mproduct(eservice)
```

Reply:
Nope it's
```suggestion
    tenantManagementService.getTenant(eservice.producerId).tupleLeft(eservice)
```

Reply:
Definitely 🆒 ! I'll try